free/videoConfig/templates/videoConfig.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'free/base.html' %}

```

Comment:
This crashes the app for me (as I don't have video configured). We need to load the page and display some error there. 

I will try the admin integration as well...